src/api/Config/index.ts

```diff
@@ -30,7 +30,7 @@ export class ConfigAPI extends Request {
    */
   public updateConfiguration(params: IUpdateConfigParams) {
     return this.request({
-      errors: [UNAUTHORIZED, FORBIDDEN],
+      errors: [UNAUTHORIZED, FORBIDDEN, INVALID_PARAMETER],
```

Comment:
@maxime-allex 
`INVALID_PARAMETER` is one of the possible errors you might get if you try to update a configuration with a wrong value

Reply:
You are the reviewer of this one @mzorkany. Why did you not approved it? This should be merged asap!